packages/ERTP/src/paymentLedger.js

```diff
@@ -434,26 +433,41 @@ export const preparePaymentLedger = (
     },
   });
 
+  /**
+   * @template {any} V
+   * @param {Zone} zone
+   * @param {string} key
+   * @param {(k: string) => V} makeValue
+   * @returns {V}
+   */
+  const provideFromZone = (zone, key, makeValue) => {
```

Comment:
@michaelfig , what do you think of adding an equivalent method to Zone, and calling it `once`?

Reply:
Without the extra level of indirection, which would then be unneeded.

Reply:
See https://github.com/Agoric/agoric-sdk/pull/7663

Reply:
Done!